### DISCUSSION TITLE
I can't seem to change the "accept" keymap for zbirenbaum/copilot.lua

### DISCUSSION THREAD

**Discussion by sdondley** [LazyVim/LazyVim#1337] [Q&A] [answered]
I've enabled copilot according to the docs here: https://www.lazyvim.org/extras/coding/copilot

Here's my config for the plugin:

```
    "zbirenbaum/copilot.lua",
    cmd = "Copilot",
    build = ":Copilot auth",
    opts = {
      suggestion = {
        enabled = true,
        auto_trigger = true,
        keymap = {
          accept = "<m-l>",     -- <<<< can't change this without running into problems
          accept_word = false,
          accept_line = false,
          next = "<M-]>",
          prev = "<M-[>",
          dismiss = "<C-]>",
        },
      },
      filetypes = {
        markdown = false,
        lua = true,
        perl = true,
        c = true,
        help = false,
      },
    },
  },
```

It works.

But I want to change the "accept" key to something else that begins with the control character. I tried setting it to `<c-/>`, `<c-m>`I run into problems. I can't accept anything for copilot after they change and it seems to also interfere with some completion  or snippet module and changes what it accepts for completion or breaks snippets all together. 

I'm editing lua files at the moment. I don't have anything in my config files for nvim-cmp or LuaSnip.

Anybody know what's going on and how to change this keymap?

**Comment by jyuan0** [comment, marked as answer]
I think you're going down the wrong path here. When you enabled the Copilot LazyVim module, [copilot-cmp](https://github.com/zbirenbaum/copilot-cmp) was also [installed](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/coding/copilot.lua#L55) and configured. I'm guessing that the Copilot suggestions you're seeing are in the [nvim-cmp](https://github.com/hrsh7th/nvim-cmp) menu, and if so, you should configure the mappings through nvim-cmp.

Also, the `suggestion` module shouldn't be enabled according to the copilot-cmp docs:
> It is recommended to disable copilot.lua's suggestion and panel modules, as they can interfere with completions properly appearing in copilot-cmp. To do so, simply place the following in your copilot.lua config:
> ```lua
> require("copilot").setup({
>   suggestion = { enabled = false },
>   panel = { enabled = false },
> })
> ```

This was already done by default when you enabled the Copilot LazyVim module.

Hope this helps.

**Reply by sdondley** [reply]
Ah, I didn't catch that zbirenbaum/copilt-cmp was not the same zbirenbaum/copilot. Confusing. I'll go back to the drawing board. Thanks.
